pkg/controller/modeladapter/modeladapter_controller.go

```diff
@@ -64,6 +65,8 @@ const (
 	ModelAdapterPodTemplateLabelKey   = "adapter.model.aibrix.ai/enabled"
 	ModelAdapterPodTemplateLabelValue = "true"
 
+	DefaultResyncInterval = 30 * time.Second
```

Comment:
@Jeffwan resync interval kept every 30s. It means, if lora_model evict by vllm. At max it will take 30 sec + Reconcile() execution time to bring lora_model back on pod. Let me know if we want to increase/decrease interval.

Reply:
I kind of feel the interval is too long. Let's change back to 10s at this moment